frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -18,37 +20,58 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
   });
 
   const handleFormSubmit = useCallback(
-    (event_: React.BaseSyntheticEvent): void => {
-      void handleSubmit(onSubmit)(event_);
+    (event: React.BaseSyntheticEvent): void => {
+      void handleSubmit(onSubmit)(event);
     },
     [handleSubmit, onSubmit],
   );
 
   return (
     <>
-      <h3>Sign Up</h3>
-      <form onSubmit={handleFormSubmit}>
+      <form onSubmit={handleFormSubmit} className={styles['auth-form']}>
+        <h1 className={styles['greeting']}>
+          Create a free account to save your preferences
+        </h1>
         <p>
           <Input
             type="text"
-            label="Email"
-            placeholder="Enter your email"
-            name="email"
+            label="Name"
+            placeholder="Please write your name"
```

Comment:
let's leave previous placeholder

Reply:
should i also leave previous placeholder for password?

Reply:
done